recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Support latest gtest
+    # https://github.com/msgpack/msgpack-c/commit/4a1db60af8d6bdb258a34352f5cd8d9800544372.patch
+    - gtest.patch
+    # This is needed for a bug in gcc version used where isnan is ambiguous. (::isnan and std::isnan)
+    - ambiguous_nan_inf.patch   # [linux]
```

Comment:
This bug is not there when using gcc 4.9 or above, but since we are using gcc 4.8 it is there.
This patch doesn't work with osx, so I'll have to find a way to make it work on all platforms before sending it upstream.